bpfd/src/dispatcher_config.rs

```diff
@@ -1,38 +1,34 @@
 // SPDX-License-Identifier: (MIT OR Apache-2.0)
 // Copyright Authors of bpfd
 
-#![no_std]
-
 // XDP Defines
-pub const XDP_METADATA_SECTION: &str = "xdp_metadata";
-pub const XDP_DISPATCHER_VERSION: u32 = 1;
-pub const XDP_DISPATCHER_RETVAL: u32 = 31;
-pub const MAX_DISPATCHER_ACTIONS: usize = 10;
+// pub (crate) const XDP_METADATA_SECTION: &str = "xdp_metadata";
```

Comment:
Nit Remove these? (and the TC ones)

Reply:
They are there for a reason I promise 😆 
It's to jog my memory when I'm working on #319

Reply:
Sounds good!